### PR TITLE
Finish Setting up the Model. 

### PR DESCRIPTION
**Pull request by Dcbaas**
Commit MSG: 
"Put classes in package folders and created all classes listed on Sant…
Signed-off-by: David <bcdave.301@gmail.com>"

I've added all of the classes Santiago and I thought of after class today. I've also subdivided the classes into package folders according to what function they serve (eg. all classes related to making the board are in the folder BoardPackage). 

For this pull request, I have not added Dustin since he hasn't been taught how this works. 

-David Baas

### CODE REVIEW

File: Model/src/CardPackage/Card.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package CardPackage;
+
+public class Card {
+}
```
Comment (kernohad) [inline Model/src/CardPackage/Card.java]: Do we want an overall Card class or ChanceCard. Isnt there multple types of cards?

File: Model/src/CardPackage/Deck.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package CardPackage;
+
+public class Deck {
+}
```
Comment (kernohad) [inline Model/src/CardPackage/Deck.java]: Same thing as Card Class, do we want a Deck class or a ChanceSquare that builds and holds an array of ChanceCard objects?

Comment (Dcbaas) [reply Model/src/CardPackage/Deck.java]: This is just a class that is used to construct a deck of cards. In the main program, there would be two instances of Deck. 

Example

private Deck ChanceCardDeck;

private Deck CommunityCheckCardDeck;

File: Model/src/CardPackage/Card.java (hunk 1 above)
Comment (Dcbaas) [reply Model/src/CardPackage/Card.java]: There are two types of cards (Chance and Community Chest) but at the end of the day, both card types do the exact same thing: Get drawn from a pile and then trigger an action in game. When I made this card class, I made it on the assumption that we could just get away with having an instance variable inside the class that determines what type of Card it is.

**Comment by kernohad** [comment]
I am closing this, although we may decide to make a design change later. Mostly doing this to test slack/git integration.

**Pull request closed without merging**